game/templates/game/board.html

```
@@ -399,6 +399,31 @@
         </div>
     </div>
 
+ <!-- ========== LEAVE GAME CONFIRMATION MODAL ========== -->
+<div class="promo-overlay" id="leaveConfirmOverlay"
+     role="dialog"
+     aria-modal="true"
+     aria-labelledby="leaveConfirmTitle">
+     style="display:none;">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)